src/pipeformer/internal/templates/inputs.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Logic for building the inputs stack template."""
+from troposphere import Output, Parameter, Ref, Tags, Template, kms, secretsmanager, ssm
+
+from pipeformer.internal.structures import Config, Input
+from pipeformer.internal.util import reference_name, resource_name
+
+from . import project_tags
+
+__all__ = ("build",)
+
+
+def _secret_value(resource: Input, tags: Tags, cmk_arn: Ref) -> secretsmanager.Secret:
+    """Construct a Secrets Manager secret to store the input value.
+
+    :param resource: Input for which to create secret
+    :param tags: Tags to set on secret
+    :param cmk_arn: Key with which to protect secret
+    :return: Constructed resource
+    """
+    return secretsmanager.Secret(
+        resource_name(secretsmanager.Secret, resource.name), KmsKeyId=cmk_arn, SecretString="REPLACEME", Tags=tags
```

Comment:
Minor quibble about using `"REPLACEME"`: this is an arbitrary value that will be swapped out as Pipeformer runs and puts in the real user inputs, so couldn't it be more specific to 1) the data being populated (e.g. `"SECRETS_MANAGER_SECRET_PLACEHOLDER"`) and/or 2) structured to be more clearly computer-readable placeholder input (e.g. `"@@PLACEHOLDER@@"`)?

This could let you do a sort of type checking/validation that the input value is being swapped for the placeholder token expected and add some error checking bounds around token substitution.

Reply:
Created #23 to track.